core/contracts/oracle/implementation/Voting.sol

```diff
@@ -461,7 +458,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
     // prettier-ignore
     function retrieveRewards(address voterAddress, uint roundId, PendingRequest[] memory toRetrieve)
         public
-        override 
+        override
         returns (FixedPoint.Unsigned memory totalRewardToIssue)
```

Comment:
nit: does this one also need to be removed?

Reply:
This blank space being removed was from the linter.

Reply:
Does the `totalRewardToIssue` variable on this line need to be removed, though? Why was it left out?

Reply:
I understand, sorry. that was an oversight. has been fixed and changes pushed.